### DISCUSSION TITLE
How to set a background image from your local drive

### DISCUSSION THREAD

**Discussion by beolse** [Akifyss/obsidian-border#139] [Q&A]
Hello,
thank you for your work:
- I applied Presets for Border theme / Paper-like,
- in Paper-like.json there is a line `"Appearance-light@@background-underlying-CSS-light": "#ffffff url(\"https://www.transparenttextures.com/patterns/cream-paper.png\" )"`,
- tell me the code, I want to set a background image from my local disk,
- (((- I can’t understand, I tried 20 options,
- Thank you

**Comment by Akifyss** [comment]
Sorry, it seems like obsidian does not allow loading local resources: https://forum.obsidian.md/t/loading-local-files-in-css-snippet/47687

However, you can choose to upload your local images to the cloud and reference the link.
you can also consider converting the local image to base64 format: https://www.base64-image.de/

**Comment by beolse** [comment]
I'm disappointed that there is no direct way to do this.
- installed the Dynamic Background Plugin -- https://github.com/samuelsong70/obsidian-dynamic-background
- there is a background with a relative file path, but it only works in the Editor area
- I would like (1) to use only your theme, (2) to have the background everywhere
- I will be grateful if you look at his code in the `background Image File`, there are 4 matches in `main.js`, and 1 in `data.json`
- unfortunately, I can't program
- Thank you for your time

**Comment by Akifyss** [comment]
This plugin doesn't work correctly in my environment and I can't test it. Sorry
